include/dynd/kernels/string_concat_kernel.hpp

```diff
@@ -14,27 +15,6 @@
 namespace dynd {
   namespace nd {
 
```

Comment:
Okay, so you probably only need one of {"#include <dynd/string.hpp>", "#include <dynd/types/string_type.hpp>"} and "#include <dynd/kernels/base_kernel.hpp>" here. We normally don't include kernel_builder directly.